monkey/infection_monkey/config.py

```diff
@@ -161,6 +161,10 @@ def as_dict(self):
 
     keep_tunnel_open_time = 60
 
+    # Monkey files directories
```

Comment:
If you're adding these new values to the config, I think you have to do it on the island config as well.
I know they aren't really configurable but I don't think this is the right solution.
I have several suggestions for this, but I think for now you should treat them the same as regular config values. In the description of the values you can add a note that they shouldn't be changed

Reply:
I don't understand if you want these directory paths to be configurable from UI or not. Is so, then I should create dirs only after monkey retrieves config from server. If not, then what's the point in adding them to island config if they're not used?